<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cha_zhao</class>
 <widget class="QWidget" name="Cha_zhao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cha_zhao</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>162</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查  找  学  生</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>294</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>246</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sou_suo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string>查    找</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="xx_gender">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>158</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="xx_id">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="xx_age">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>206</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>202</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="xx_name">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QListView" name="student_ti">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查  找  学  生</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ti_ti34">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>age_2</zorder>
  <zorder>gender_2</zorder>
  <zorder>sou_suo</zorder>
  <zorder>xx_gender</zorder>
  <zorder>name_2</zorder>
  <zorder>xx_id</zorder>
  <zorder>xx_age</zorder>
  <zorder>label_11</zorder>
  <zorder>id_2</zorder>
  <zorder>xx_name</zorder>
  <zorder>student_ti</zorder>
  <zorder>ti_ti34</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sou_suo</sender>
   <signal>clicked()</signal>
   <receiver>Cha_zhao</receiver>
   <slot>sou_sou()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xx_name</sender>
   <signal>clicked()</signal>
   <receiver>Cha_zhao</receiver>
   <slot>xx_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xx_id</sender>
   <signal>clicked()</signal>
   <receiver>Cha_zhao</receiver>
   <slot>xx_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xx_gender</sender>
   <signal>clicked()</signal>
   <receiver>Cha_zhao</receiver>
   <slot>xx_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>767</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xx_age</sender>
   <signal>clicked()</signal>
   <receiver>Cha_zhao</receiver>
   <slot>xx_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>838</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>905</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sou_sou()</slot>
  <slot>xx_1()</slot>
  <slot>xx_2()</slot>
  <slot>xx_3()</slot>
  <slot>xx_4()</slot>
 </slots>
</ui>
